test/unit/helpers/xhr/PIV_CHALLENGE.js

```diff
@@ -0,0 +1,7 @@
+define({
```

Comment:
should this file be named just `PIV_POST`? we are not in an mfa_challenge status or similar, are we?